packages/blade/src/components/Input/BaseInput/BaseInputTagSlot.web.tsx

```diff
@@ -96,10 +99,11 @@ const BaseInputTagSlot = ({
     tags,
     maxTagRows,
     visibleTagsCountRef,
+    showAllTags,
   });
 
   React.useEffect(() => {
-    slotRef.current?.scrollTo?.({
+    slotRef.current?.scrollTo({
```

Comment:
Why did we remove the check for scrollTo? We sure it won't ever fail?

Reply:
yup that referenced element is in the same file only and we have overflow there.

Reply:
oh I remembered why I added that earlier. It fails in tests because jsdom can't scroll. Added it back just to on safer side anyway.